src/Button/index.js

```diff
@@ -134,16 +138,32 @@ export const StyledButton = styled(
           : tokens.colorTextButtonActive[type])};
     }
   }
+
+  ${StyledSpinner} {
+    width: ${({ tokens, size }) => tokens.loadingWidth[size]};
+    height: ${({ tokens, size }) => tokens.loadingHeight[size]};
+  }
 `;
 
 StyledButton.defaultProps = {
   theme: defaultTokens,
 };
 
+const StyledButtonContent = styled.div`
+  visibility: ${({ loading }) => loading && "hidden"};
+  height: 100%;
+  display: flex;
```

Comment:
Why flex here ?

Reply:
I added a new wrapper where is necessary to pass flex here. Without it will content of the Button broken.